points_tracker/views.py

```diff
@@ -111,24 +112,24 @@ def upload_file():
                     pass
                 file.save(filepath)
 
+                #load the file
                 if extension == ".mp3":
-                    length = int(round(MP3(filepath).info.length))
-                    sound = AudioSegment.from_mp3(filepath)
-                    sound.export(filepath, format="wav")
-                elif extension == ".mp4":
-                    length = int(round(MP4(filepath).info.length))
-                    sound = AudioSegment.from_file(filepath)
-                    sound.export(filepath, format="wav")
+                    audiofile = AudioSegment.from_mp3(filepath)
                 elif extension == ".wav":
-                    length = 0
+                    audiofile = AudioSegment.from_wav(filepath)
                 else:
-                    length = 0
+                    audiofile = AudioSegment.from_file(filepath)
```

Comment:
Why not replace this if with 

    audiofile = AudioSegment.from_file(filepath, filepath.split(".")[-1])

?